src/applications/vaos/services/location/transformers.js

```diff
@@ -163,6 +162,19 @@ function getTestFacilityId(facilityId) {
   return facilityId;
 }
 
+/**
+ * Returns whether or not the facility has a COVID vaccine phone line
+ *
+ * @export
+ * @param {Object} facility A facility resource
+ * @returns {Boolean} Whether or not the facility has a COVID vaccine phone line
+ */
+export function hasCovidPhoneNumber(facility) {
```

Comment:
I think you can leave off `export` here (and the `@export` comment above), since it's only used in this file.